packages/canary-react/src/component-tests/IcDateInput/IcDateInput.cy.tsx

```diff
@@ -691,12 +692,24 @@ describe("IcDateInput", () => {
     cy.findShadowEl(DATE_INPUT, "#clear-button").shadow().find("button").focus().click()
     
     cy.findShadowEl(DATE_INPUT, MONTH_INPUT_ARIA_LABEL).should(
-      "have.value",
+      HAVE_VALUE,
       ""
     ); 
 
     cy.get("@icDateChanged").should((stub) => {
       expect(stub.getCall(0).args[0].detail.value).to.equal(null);
     });
   });
+
+  it.only("should update helper text and reset helper text to default when set to empty string", () => {
```

Comment:
```suggestion
  it("should update helper text and reset helper text to default when set to empty string", () => {
```

Reply:
Good spot, feeling silly for checking that in...